src/Features/Core/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.GlobalSuppressMessageCodeAction.cs

```diff
@@ -41,13 +41,8 @@ public GlobalSuppressMessageCodeAction(AbstractSuppressionCodeFixProvider fixer,
                 };
             }
 
-            public override string Title
-            {
-                get
-                {
-                    return _title;
-                }
-            }
+            public override string Title => _title;
+            public override string EquivalenceKey => _title + _diagnostic.Id + _targetSymbol.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat);
```

Comment:
Actually, I was thinking of even getting rid of targetSymbol completely from the equivalence key here, otherwise the FixAll for GlobalSuppressMessage (which we hope to support soon) will be a no-op. I think the only realistic case where you could have multiple target symbol declarations on the same line is for fields, and it should be fine to offer a single fix for a single field with a FixAll experience, rather then offer same Suppress fixes for every field declared on the line. Sounds reasonable?